Tests/SwiftMockTests/PropertyStubbingTests.swift

```diff
@@ -29,9 +30,11 @@ protocol AsyncThrowsGetPropertyProtocol {
 final class PropertyStubbingTests: XCTestCase {
 	override func setUp() {
 		continueAfterFailure = false
-		testFailureReport = { message in
-			XCTFail(message)
-		}
+		SwiftMockConfiguration.setUp()
+	}
+	
+	override func tearDown() {
+		SwiftMockConfiguration.tearDown()
```

Comment:
`super.tearDown()` is not called here, is that intentional?

Reply:
No, but it don't change anything in this case, so it's not very important